npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher/path_dependency_builder.rb

```diff
@@ -118,13 +118,30 @@ def parsed_yarn_lock
           # to Yarn, which may not be safe
           @parsed_yarn_lock ||=
             begin
+              # Transform lockfile to parseable YAML by wrapping requirements in
+              # quotes, e.g. ("pkg@1.0.0":) and adding colon to nested
+              # properties (version: "1.0.0")
+              sanitize_requirement = lambda do |line|
```

Comment:
We should probably split this (ghetto Yarn.lock parsing) out into a separate class. If you've got loads on your plate I'm 👍 on keeping it here for now, though.

Reply:
Done!